fuzz/fuzz_targets/differential_spec.rs

```diff
@@ -0,0 +1,57 @@
+#![no_main]
+
+use libfuzzer_sys::arbitrary::{Result, Unstructured};
+use libfuzzer_sys::fuzz_target;
+use std::sync::atomic::{AtomicUsize, Ordering::SeqCst};
+use wasmtime_fuzzing::{generators, oracles};
+
+// Keep track of how many WebAssembly modules we actually executed (i.e. ran to
+// completion) versus how many were tried.
+static TRIED: AtomicUsize = AtomicUsize::new(0);
+static EXECUTED: AtomicUsize = AtomicUsize::new(0);
+
+fuzz_target!(|data: &[u8]| {
+    // errors in `run` have to do with not enough input in `data`, which we
+    // ignore here since it doesn't affect how we'd like to fuzz.
+    drop(run(data));
+});
+
+fn run(data: &[u8]) -> Result<()> {
+    let mut u = Unstructured::new(data);
+    let mut config: generators::Config = u.arbitrary()?;
+    config.module_config.set_differential_config();
+
+    // Enable features that the spec interpreter has implemented
+    config.module_config.config.bulk_memory_enabled = false;
+    config.module_config.config.reference_types_enabled = false;
+    config.module_config.config.simd_enabled = false;
+    config.module_config.config.relaxed_simd_enabled = false;
+    config.module_config.config.exceptions_enabled = false;
+    config.module_config.config.module_linking_enabled = false;
```

Comment:
These should be safe to omit setting to `false` here since we can rely on `set_differential_config` above setting them all to false, although setting `multi_value_enabled` to false below would need to stay since `set_differential_config` doesn't specifically set that one to `false`